cypress/plugins/index.js

```diff
@@ -8,7 +7,6 @@ const getCypressEnvVariables = require('./getCypressEnvVariables');
 const path = require('path');
 
 module.exports = (on, config) => {
-    networkShim(on);
```

Comment:
@simonadomnisoru I don't think we need this, but I might be wrong. Any insights?

Reply:
You're right. We don't need it. We use live network requests therefore there is no need to record them and play them back.